config/testgrids/kubernetes/sig-network/config.yaml

```diff
@@ -87,6 +87,12 @@ dashboards:
       description: network gci-gce with kube-dns serial e2e tests for master branch
       alert_options:
         alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+    - name: gci-gce-serial-kube-dns-nodecache
+    test_group_name: ci-kubernetes-e2e-gci-gce-serial-kube-dns-nodecache
+    base_options: include-filter-by-regex=\[sig-network\]
```

Comment:
I think you might have to indent one further for this to parse correctly, though it should fail tests if that's true.